src/interactive/azext_interactive/azclishell/configuration.py

```diff
@@ -73,7 +70,7 @@ def __init__(self, cli_config, style=None):
                 self.config.write(config_file)
         else:
             with open(os.path.join(self.config_dir, CONFIG_FILE_NAME), 'r') as config_file:
-                self.config.readfp(config_file)  # pylint: disable=deprecated-method
```

Comment:
`readfp` is deprecated since Python 3.2. I guess use it to support Python 2.7.

https://docs.python.org/3.11/library/configparser.html#configparser.ConfigParser.readfp

Reply:
> `readfp` is deprecated since Python 3.2. I guess use it to support Python 2.7.
> 
> https://docs.python.org/3.11/library/configparser.html#configparser.ConfigParser.readfp

I thought we no longer needed to support Python 2?

Reply:
Yes, it's been EOL for almost five years.